__tests__/UITests.tsx

```diff
@@ -335,7 +335,7 @@ describe("LibraryContainer UI", function () {
     }, 500);
   });
 
-  it("coregroup items should auto expand", function () {
+  xit("coregroup items should auto expand", function () {
```

Comment:
Is it possible to test add-on elements, seems they are not currently in the json so probably not doable right now?

Reply:
yup...now I am trying to add a json to this and see whether a unit test is possible.

Reply:
@QilongTang Added a test :)

Reply:
Thank you @ramramps Looks good